notebooks/00-release-28.0.0/03-lookup-tables-28.ipynb

```diff
@@ -0,0 +1,1129 @@
+    "FROM \"example-flights-lookup\" a\n",
+    "LEFT JOIN lookup.\"example-flights-airportsizes\" b ON a.Origin = b.k\n",
+    "WHERE TIME_IN_INTERVAL(__time,'2005-11-30T11:00:00/2015-11-30T08:00:00')\n",
+    "GROUP BY 1\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3b35b59e-503d-4a00-bf49-97781f9756d7",
+   "metadata": {},
+   "source": [
+    "### Return lookup values using a function\n",
+    "\n",
+    "Use the LOOKUP [string function](https://druid.apache.org/docs/latest/querying/sql-scalar#string-functions) to dereference keys to values in an SQL query directly. This function is available as both a [SQL](https://druid.apache.org/docs/latest/querying/sql-functions#lookup) and [native](https://druid.apache.org/docs/latest/querying/math-expr#string-functions) expression.\n",
+    "\n",
+    "In the following cell, the `Origin` key from the `example-flights-lookup` table is used to match against the `\"example-flights-airportsizes\"` lookup, returning the value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e58653fa-54a9-439a-8461-b95e49ce9daa",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "SELECT\n",
+    "    LOOKUP(\"Origin\", 'example-flights-airportsizes') AS \"airportSize\",\n",
+    "    COUNT(DISTINCT Origin) AS \"airports\",\n",
+    "    COUNT(*) AS \"flights\",\n",
+    "    SUM(Distance) AS \"totalDistance\"\n",
+    "FROM \"example-flights-lookup\"\n",
+    "WHERE TIME_IN_INTERVAL(__time,'2005-11-30T11:00:00/2015-11-30T08:00:00')\n",
+    "GROUP BY 1\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8bbb8830-c0e9-45d7-9533-4f0c65144367",
+   "metadata": {},
+   "source": [
+    "Use the overloaded form of the LOOKUP function to return a value even when there is no matching key.\n",
+    "\n",
+    "> This version of the LOOKUP function requires Druid version 28.0.0 or above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1189853e-9a7c-4bea-8997-bcb5578fd01a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "SELECT\n",
+    "    LOOKUP(\"Origin\", 'example-flights-airportsizes', 'Not found') AS \"airportSize\",\n",
+    "    COUNT(DISTINCT Origin) AS \"airports\",\n",
+    "    COUNT(*) AS \"flights\",\n",
+    "    SUM(Distance) AS \"totalDistance\"\n",
+    "FROM \"example-flights-lookup\"\n",
+    "WHERE TIME_IN_INTERVAL(__time,'2005-11-30T11:00:00/2015-11-30T08:00:00')\n",
+    "GROUP BY 1\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d9966aab-4965-4b75-bc42-209d846050df",
+   "metadata": {},
+   "source": [
+    "## Query multiple lookups\n",
+    "\n",
+    "Run the following cell to create a new lookup table, again using a `map` to inline the data direct to the API.\n",
```

Comment:
```suggestion
    "Run the following cell to create a new lookup table using a `map` to correlate the data direct to the API.\n",
```
"again" reads awkwrdly here. Is "inline" a verb? Prefer something more direct if possible